File: 78 Sine Wave/javascript/sinewave.js
print(tab(30), "SINE WAVE");
print(tab(15), "CREATIVE COMPUTING  MORRISTOWN, NEW JERSEY");
print("\n\n\n\n");

// REMARKABLE PROGRAM BY DAVID AHL
// Transliterated to Javascript by Les Orchard <me@lmorchard.com>

let toggleWord = true;

for (let step = 0; step < 40; step += 0.25) {
  let indent = Math.floor(26 + 25 * Math.sin(step));
  print(tab(indent), toggleWord ? "CREATIVE" : "COMPUTING");
  toggleWord = !toggleWord;
}

function print(...messages) {
  console.log(messages.join(" "));
}

function tab(count) {
  return " ".repeat(count);
}

File: 78 Sine Wave/perl/sinewave.pl
#!/usr/bin/perl
use strict;
use warnings;

print ' ' x 30 ."SINE WAVE\n";
print ' ' x 15 ."CREATIVE COMPUTING  MORRISTOWN, NEW JERSEY\n";
print "\n\n\n\n\n";

my $B=0;

for (my $T=0; $T<40; $T+=.25) {
	my $A=int(26+25*sin($T));
	print ' ' x $A;
	if ($B==0) { print "CREATIVE\n"; }
	if ($B==1) { print "COMPUTING\n"; }
	$B= !$B; #Toggle
	}
